pkg/utils/metricutil/metricutil.go

```diff
@@ -102,3 +105,28 @@ func instanceName() string {
 	}
 	return hostname
 }
+
+// EnablePyroscope enables pyroscope if pyroscope is enabled.
+func EnablePyroscope() {
+	if os.Getenv("PYROSCOPE_SERVER_ADDRESS") != "" {
```

Comment:
Do we need to use config? I think no user will use env to enable this feature🤔


Ref https://github.com/pingcap/tidb/pull/52139#discussion_r1550572098

Reply:
Only keep the same with the tidb implement. I think this feature may cause some possible performance regression.

Reply:
It is only used by internal profilers. It is convenient for pingcap  to find performance problems.